### PR TITLE
Improve the text for creating a profile

### PR DESCRIPTION
**Pull request by lukaw3d**
| Before | After |
| --- | --- |
|  before | localhost_3000_create-wallet(extension375x600 + scrollbar)  |

Mocks:
image

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -391,10 +391,11 @@
   },
   "persist": {
     "createProfile": {
-      "choosePassword": "Choose a password",
       "passwordMismatch": "Entered password does not match",
-      "repeatPassword": "Re-enter your password",
-      "startPersisting": "Store private keys locally, protected by a password",
+      "repeatPassword": "Confirm your password",
+      "startPersisting": "Store your private keys locally and protect them with a password by creating a profile.",
```
Comment (lubej) [inline src/locales/en/translation.json:396]: "Securely store your private keys on your local device, protected by password."?

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -391,10 +391,11 @@
...
+      "startPersistingToggle": "Create a profile",
+      "undependableAcknowledge": "I understand this password and profile do not substitute my mnemonic.",
```
Comment (lubej) [inline src/locales/en/translation.json:398]: I understand that neither password or profile serve as a substitution for my mnemonic.

**Review by lubej — COMMENTED** [review]
Would be nice if we get someone from marketing(native speaker) to go over the texts.

**Review by lubej — APPROVED** [review]

File: src/locales/en/translation.json (hunk 2 above)
Comment (lukaw3d) [reply src/locales/en/translation.json:398]: cc @donouwens do you want to iterate more on it

Comment (lukaw3d) [reply src/locales/en/translation.json:398]: (i'm merging in current state just to avoid code conflict, and will revisit)

**Pull request merged**